libs/react-rx-store/tsconfig.json

```diff
@@ -1,15 +1,20 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "jsx": "react",
     "allowJs": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "types": ["node", "jest"]
   },
-  "files": [
-    "../../node_modules/@nrwl/react/typings/cssmodule.d.ts",
-    "../../node_modules/@nrwl/react/typings/image.d.ts"
-  ],
-  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
+  "files": [],
+  "include": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
```

Comment:
Whats tsconfig.lib.json meant for?

Reply:
See above. This is all codemod changes made by nx. https://github.com/rx-store/rx-store/pull/36#discussion_r463988443

Reply:
Hm, ideally I'd like to understand why they're doing this, as it does potentially complicate debugging the whole mono-repo setup if something goes wrong later on, given that there's more project references. It'd be nice to know why this change specifically